tests/Microsoft.Bot.Builder.LanguageGeneration.Renderer.Tests/LGGeneratorTests.cs

```diff
@@ -200,5 +200,7 @@ private TestFlow CreateFlow(string locale, BotCallbackHandler handler)
 
             return new TestFlow(adapter, handler);
         }
+
+        private readonly ImportResolverDelegate DefaultResolver = LanguageGeneratorManager.ResourceResolver(resourceExplorer);
```

Comment:
call this resourceResolver? default is confusing me, i thought it was fileResolver